model/publishing/delivery/PublishingDeliveryService.php

```diff
@@ -51,140 +40,4 @@ class PublishingDeliveryService extends ConfigurableService
     const DELIVERY_REMOTE_SYNC_COMPILE_ENABLED = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#ComplyEnabled';
 
     const DELIVERY_REMOTE_SYNC_REST_OPTION = 'remote-publish';
-
-    /**
-     * @param \core_kernel_classes_Resource $delivery
-     * @return \common_report_Report
-     * @throws \common_exception_Error
-     * @throws \common_exception_NotFound
-     * @throws \core_kernel_persistence_Exception
-     */
-    public function publishDelivery(\core_kernel_classes_Resource $delivery)
```

Comment:
Removing public methods from a service is definitely a breaking change.
Shall we, perhaps, bump a major version in the `manifest`?

Reply:
You are right, updated.